<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transfer</class>
 <widget class="QDialog" name="transfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="verify_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>12</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="toacct_label">
        <property name="text">
         <string>账户：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="toacct_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="amount_label">
        <property name="text">
         <string>金额：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="amount_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verify_button</sender>
   <signal>clicked()</signal>
   <receiver>transfer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
